clusters/nerc-ocp-edu/gatekeeper-policy/application.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: gatekeeper-policy
+  labels:
+    nerc.mghpcc.org/sync-policy: common
+spec:
+  destination:
+    name: nerc-ocp-edu
+    namespace: gatekeeper-system
+  project: default
+  source:
+    repoURL: https://github.com/ocp-on-nerc/gatekeeper.git
+    path: policy/overlays/nerc-ocp-edu
+    targetRevision: HEAD
```

Comment:
This overlay is missing: https://github.com/OCP-on-NERC/gatekeeper/tree/main/policy/overlays/nerc-ocp-edu